### PR TITLE
Bug Cleanups

### PR DESCRIPTION
**Pull request by zachgibb**

### CODE REVIEW

**Comment by zachgibb** [comment]
- closes #132
- closes #127 
- closes #109

**Comment by zachgibb** [comment]
bcde468 fixes #137

File: src/screens/SendRequest/ContactModal.js [hunk 1]
```diff
@@ -253,6 +247,15 @@ export default class ContactModal extends Component {
                   />
                 )}
               />
+              <LinearGradient
+                end={{ x: 1, y: 1 }}
+                start={{ x: 1, y: 0 }}
+                colors={[
+                  'rgba(61,67,74,0)',
```
Comment (ericgibb) [inline src/screens/SendRequest/ContactModal.js]: could this be abstracted for use elsewhere? maybe add it as an exported array from `styles/index.js`

Comment (zachgibb) [reply src/screens/SendRequest/ContactModal.js]: fixed by baac382 in PR #140

**Pull request merged**